src/transforms/transform-pull-request.ts

```diff
@@ -30,13 +30,12 @@ const mapReviews = async (
 	gitHubInstallationClient: GitHubInstallationClient
 ): Promise<JiraReview[]> =>
 {
-	const sortedReviews = orderBy(reviews, "submitted_at", "desc");
```

Comment:
Turns out we don't want to sort the reviews. When GH sends us the reviews they're sending them in the order we want. By reversing them, we wind up sending the original review instead of the most recent.

Reply:
Otherwise reader might wonder why we don't do `asc` sorting instead.

Reply:
The definition of chronological order is: "Chronological order refers to the arrangements of events according to the time they occurred (i.e., from first to last)." In the case of reviews, this means GH sends us the reviews starting from the first through to the last. So it's then the last review that we check for each reviewer and ship off to Jira.

Previously, GH was sending these from first to last but then we were flipping them around from last to first. This meant we mapped over all the reviews and sent Jira the review state of the **first** review for each reviewer. 

We don't need to explicitly sort by ascending because that's how the data is already sent to us from GH.